brianlong/www.validators.app
# 188610980 map with data centers

#### What's this PR do?
- provides a map with data centers

#### How should this be manually tested?
- go to /map/index and check the map for errors
- check if all the functions work as intended
- check if there are no errors in console

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/188610980)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
